aiida/schedulers/datastructures.py

```diff
@@ -248,6 +248,8 @@ class JobTemplate(DefaultFieldsAttributeDict):  # pylint: disable=too-many-insta
       * ``rerunnable``: if the job is rerunnable (boolean)
       * ``job_environment``: a dictionary with environment variables to set
         before the execution of the code.
+      * ``envvar_double_quotes``: if true, use double quotes to escape the environment
+        variables in bash script.
```

Comment:
```suggestion
      * ``envvar_double_quotes``: if set to True, use double quotes instead of single quotes to escape the environment variables specified in ``environment_variables``.
```